src/transfer-manager.ts

```diff
@@ -19,8 +19,12 @@ import {DownloadOptions, DownloadResponse, File} from './file';
 import * as pLimit from 'p-limit';
 import * as path from 'path';
 import * as extend from 'extend';
-import {promises as fsp} from 'fs';
+import {createReadStream, promises as fsp} from 'fs';
 import {CRC32C} from './crc32c';
+import {GoogleAuth} from 'google-auth-library';
+import {XMLParser, XMLBuilder} from 'fast-xml-parser';
```

Comment:
Add to package JSON? It would be interesting to see the level of effort for adding this to one of our shared libs (gaxios?).

Reply:
I added it: https://github.com/googleapis/nodejs-storage/pull/2192/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R62. I'm sure we could write something ourselves. I figured since this was a one off usage of the XML API it probably wasn't worth doing ourselves.